torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4949,6 +4968,12 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                SkipInfo('TestCommon', 'test_variant_consistency_eager'),
            ),
            sample_inputs_func=sample_inputs_addbmm),
+    OpInfo('attn',
+           dtypes=floating_types_and(torch.bfloat16),
+           backward_dtypes=floating_types_and(torch.bfloat16),
```

Comment:
You don't need to specify this one if it matches the one above right?

Reply:
correct! they used to differ, hence the origin of this.